app/screens/home/RouteList.tsx

```diff
@@ -231,6 +238,11 @@ const RouteList = ({
       label: "model.route.fee",
       value: `${route.fee}%` + (route.refBonus ? ` (${route.refBonus}% ${t("model.route.ref_bonus")})` : ""),
     },
+    {
+      condition: true,
+      label: "model.route.min_deposit",
+      value: `${formatMinDeposit(route.minDeposits, "BTC")}`,
+    }, // BTC hardcoded until we get asset for deposit from API
```

Comment:
I would solve this (for all routes) in a very generic way:
```ts
value: route.minDeposits.map((d) => `${d.amount} ${d.asset}`).join(' / '),
```

Reply:
fair enough, but why should we show min deposits for other blockchains for example. If we have a min deposit for ETH. Then I wouldn't show it for other crypto-routes

Reply:
IMO it's the task of the API to determine which min. deposits apply for the specific route.